### PR TITLE
adding graylog content packs

### PR DESCRIPTION
**Pull request by matusdrobuliak66**
## What do this changes do?
- adding graylog content packs
  - introduce graylog extractor (for extracting structured logs from simcore)
  - introduce graylog pipeline (for replacing new line characters for better readability)
  - introduce osparc-default & osparc-simcore-default search views (for better searching experience)

## Related issues
- https://github.com/ITISFoundation/osparc-simcore/issues/4079

### CODE REVIEW

File: services/graylog/docker-compose.yml [hunk 1]
```diff
@@ -92,3 +99,7 @@ networks:
   public:
     external: true
     name: ${PUBLIC_NETWORK}
+
+configs:
+  graylog_config:
+    file: ./data/contentpacks/simcore-content-pack.json
```
Comment (mrnicegyu11) [inline services/graylog/docker-compose.yml:105]: Missing newlines? It looks to me like you did not install and run pre-commit hooks, please do this, thx :P

**Comment by mrnicegyu11** [comment]
Looks good please run precommit hooks @matusdrobuliak66 , thanks a lot!

File: services/graylog/docker-compose.yml (hunk 1 above)
Comment (matusdrobuliak66) [reply services/graylog/docker-compose.yml]: I have run it with precommit and add some small changes on a way

**Review by mrnicegyu11 — APPROVED** [review]
superb job! thanks

File: services/graylog/docker-compose.yml [hunk 2]
```diff
@@ -53,6 +56,10 @@ services:
       # Username: admin
       - GRAYLOG_ROOT_PASSWORD_SHA2=${GRAYLOG_ROOT_PASSWORD_SHA2}
       - GRAYLOG_HTTP_EXTERNAL_URI=${GRAYLOG_HTTP_EXTERNAL_URI}
+      # Loading & install our custom content pack
+      - GRAYLOG_CONTENT_PACKS_LOADER_ENABLED=true
+      - GRAYLOG_CONTENT_PACKS_DIR=/files
+      - GRAYLOG_CONTENT_PACKS_AUTO_INSTALL=simcore-content-pack.json
```
Comment (sanderegg) [inline services/graylog/docker-compose.yml:62]: that is a nice addition. is this new from the latest version of graylog?

Comment (matusdrobuliak66) [reply services/graylog/docker-compose.yml:62]: I don't think so, I found something about them even in version 3.

**Pull request merged**